notebooks/install_ocaml_colab.ipynb

```diff
@@ -110,6 +110,7 @@
         "  run_script('(cd; tar -xzf \"%s\")' % read_cache_)\n",
         "\n",
         "run_script('echo \\'#use \"topfind\";;\\' >> /root/.ocamlinit')\n",
+        "run_script('/root/.opam/4.07.1/.opam-switch/sources/jupyter.2.7.2/config/ocaml-jupyter-opam-genspec')\n",
```

Comment:
Thanks a lot. I think we need to restrict the version of ocaml-jupyter as follows:

```
81:        "opam_packages_ = [ 'base', 'jupyter=2.7.2' ]\n",
```

Reply:
Indeed. As a future improvement, the script could check the installed version of ocaml-jupyter and invoke the script through the corresponding path.